### DISCUSSION TITLE
Add security features when used

### DISCUSSION THREAD

**Discussion by Tocily** [keepassxreboot/keepassxc#9091] [Ideas]
Because everyone knows the boot password of the office computer, I hope to add the ability to enter the main password again when displaying the password and exporting the password file.

**Comment by droidmonkey** [comment]
What does knowing the boot password have to do with KeePassXC?

**Reply by Tocily** [reply]
There is a quick database unlock in the settings. After opening, use the computer power-on password to unlock my database, and then you can view or export the password.
For the convenience of use, it is my habit to use the power-on password to unlock the database. I don’t want to enter the master password every time, which is easy to be peeped

**Reply by Tocily** [reply]
More secure if asked to re-enter the master password when viewing or exporting passwords

**Reply by Tocily** [reply]
For such an important operation as exporting a password, verifying the password is more secure.
